src/stavenote.ts

```diff
@@ -779,7 +779,7 @@ export class StaveNote extends StemmableNote {
   getModifierStartXY(
     position: number,
     index: number,
-    options: {
+    options?: {
```

Comment:
We can make this non-optional with `options: { ... } = {}`, and remove the `options = options || {}` in the body.

Reply:
done